### ISSUE TITLE
Problem displaying camera stream on google home mini

### ISSUE THREAD

**Issue by kghxkgh** [andrei-tatar/node-red-contrib-smartnora#139]
Hi Andreii,  
My user id is: xBOfphZZ5uVGdZGsyX6jIef5Kgy2

I have a nodered flow to display a camera on my google home minis.
I used docker 'streamer' to convert an rtsp stream to a .m3u8 stream.
This was working up until 1-2 weeks ago.
Now all I get is an image showing the camera trying to load. (image below)

I had to change my ISP which changed my home configuration and wifi.
I am not sure if this is related but now my camera stream is not showing on the home.

I have checked the stream is working in VLC. (url: http://192.168.0.200:8082/garden.m3u8)
I didn't change the nodered flow, but am wondering if you have any way to verify the connection.

I have just removed some nora nodes from my nodered, as I had several more than the 5 limit (just tests) and thought maybe this could be the issue.

I saw also in another bug that a reset of the device could help. I have done a factory reset for my google home and the situation is still the same.

The node in nodered indicates it is connected. (grren light)

All I see on the google home is the following screen like it is trying to load the stream.
![image](https://user-images.githubusercontent.com/74844260/147842314-186ca176-e203-402d-9f84-7ce7e969bc69.png)

I also setup a simple switch and this works fine from nodered, my phone home app and also on the google home mini.

Hoping you can help. Keith from Oz.

**Comment by andrei-tatar** [comment]
@kghxkgh Hi,
I admit I don't use the camera node but I was always under the impression that it needs to be a publicly accessible URL. I see in your case it's a local URL (192.168.x.x). 
Are the camera and the Google Hub in the same LAN? Google Home might be uncapable of connecting to that IP if it's in a different LAN (because of NAT. You might need to setup some forwarding rules).

**Comment by kghxkgh** [comment]
I had it working using a 10.10.x.x address. I now only have the one LAN using my ISP router. I will make a trace on my server where the .m3u8 is created and see if i can find the issue there.

It is likely a connection issue of some sort on my side. I was just wondering whether you can see any messages in your logs.

If there is not much to see, then I am fine for you to close the issue. 
If I can get this working again I will subscribe, as I do like the service and integration with nodered.

**Comment by andrei-tatar** [comment]
Nope, not much to see in my logs. I think Google connects directly to the camera URL.
Let us know what it was if you find the root cause.

**Comment by kghxkgh** [comment]
Thanks, it works fine when i use a public test url. I know it worked before with a 10.10.x.x url. I made a trace on my server and can see some traffic between my google home and the streamer container, but can't make much sense of it. I'll let you know the result when i find it.

**Comment by kghxkgh** [comment]
I have confirmed it is an issue in my home network, so i will close this.
Although i haven't resolved it, I have found a workaround. 

I have the following setup:
ISP modem/router provides wifi-1.
Switch connected by cable to ISP router.
AP connected by cable to Switch. Provides wifi-2.
Server connected by cable to Switch. Runs docker, nodered and streaming service.

When Google home is connected to wifi-2 the camera streaming is immediate and how i expect it to work.

When Google home (GH) is connected to wifi-1 the camera stream does not work. 
When I connect my laptop to wifi-1, the camera stream works fine from VLC.

I made several network traces from the server and can see the GH sends a packet to start the connection and is returned an ACK, but does not move on to the next packets.

Unfortunately, I am limited with options for network monitoring on the ISP router and GH device, so not sure where I can go from here. I will try moving my server onto a second port on the ISP router, and see if this helps. It is strange because everything else i have from printers, to APs, my server all work off the switch. This is the only issue found.
